src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.ILocalState.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal partial class AbstractFlowPass<TLocalState>
+    {
+        /// <summary>
+        /// This is the "top" state of the data flow lattice. Generally,
```

Comment:
is it worth documenting what is meant by 'lattice'?  I ask since it's been roughly 20 years since my last compilers course dealing with these terms.  So it might be nice to have something to point at.  A link to some reasonable wiki page would be fine as well :)